src/components/views/rooms/EntityTile.tsx

```diff
@@ -168,14 +179,12 @@ class EntityTile extends React.Component {
             e2eIcon = <E2EIcon status={e2eStatus} isUser={true} bordered={true} />;
         }
 
-        const BaseAvatar = sdk.getComponent('avatars.BaseAvatar');
-
         const av = this.props.avatarJsx ||
             <BaseAvatar name={this.props.name} width={36} height={36} aria-hidden="true" />;
 
         // The wrapping div is required to make the magic mouse listener work, for some reason.
         return (
-            <div ref={(c) => this.container = c} >
+            <div ref={this.container} >
```

Comment:
I can't see anything actually using this ref, especially as its signature is changing from a LegacyRef to a RefObject

Reply:
```suggestion
            <div ref={this.container}>
```

Reply:
Yep, it doesn't seem used